uptrain/core/classes/visuals/__init__.py

```diff
@@ -1,3 +1,3 @@
 from .abstract_visual import AbstractVisual
-from .umap import DimensionalityReduction, UMAP_PRESENT
+from .dimensionality_reduction import DimensionalityReduction, UMAP_PRESENT
```

Comment:
@Dominastorm this change is already in main. Can you merge main in this PR and push again to avoid potential conflicts?

Reply:
Yeah sure